src/NHibernate.Test/Events/Collections/AbstractCollectionEventFixture.cs

```diff
@@ -8,6 +8,7 @@
 
 namespace NHibernate.Test.Events.Collections
 {
+	[TestFixture]
 	public abstract class AbstractCollectionEventFixture : TestCase
```

Comment:
With current generation rules, it is required to flag a base class with `[TextFixture]` if it defines tests potentially needing an async counterpart.